index.html

```diff
@@ -60,6 +61,181 @@ <h2 class="heading heading2">I’m a software developer</h2>
           </div>
         </section>
       </section>
+
+      <!-- My Works -->
+
+      <section id="projects">
+        <div>
+          <h2 class="heading title">My Recent Works</h2>
+          <hr class="divider" />
+        </div>
+        <!-- Project 1 -->
+
+        <div class="works_card">
+          <div class="card_background">
+            <h3 class="card_heading">Multi-Post Stories Gain+Glory</h3>
+            <ul class="lang_container">
+              <li class="language_btn">Ruby On Rails</li>
+              <li class="language_btn">html</li>
+              <li class="language_btn">css</li>
+              <li class="language_btn">JavScript</li>
+            </ul>
+            <a href="#"><button class="button">See Project</button></a>
+          </div>
+        </div>
+
+        <!-- Project 2 -->
+
+        <div class="works_card">
+          <div class="card_background">
+            <h3 class="card_heading">Multi-Post Stories Gain+Glory</h3>
+            <ul class="lang_container">
+              <li class="language_btn">Ruby On Rails</li>
+              <li class="language_btn">html</li>
+              <li class="language_btn">css</li>
+              <li class="language_btn">JavScript</li>
+            </ul>
+            <a href="#"><button class="button">See Project</button></a>
```

Comment:
- [ ] Nesting a `button ` in `a` tag is not valid in HTML5. you can check [this link](https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5) for more details. I suggest you use one of the previous code reviewer suggestions https://github.com/Anjacodes/Portfolio/pull/2#discussion_r734066402. Kindly make the change in all the project cards.

Reply:
Hi Safa, 

Thank you for your quick response! 
Solved it like this now. Will that be ok?
<img width="339" alt="Screen Shot 2021-10-21 at 18 51 31" src="https://user-images.githubusercontent.com/75140420/138372127-bc7d215d-b6d5-47e0-9ad4-6c30d5647126.png">